src/backend/api/reservations.js

```diff
@@ -0,0 +1,71 @@
+const express = require("express");
+const { on } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+
+
+router.get("/", async (request, response) => {
+    const reservations = await knex("reservations");
+    response.json(reservations);
+})
+
+router.post("/", async (request, response) => {
+
+    const insertedReservations = await knex("reservations")
+        .insert({
+            number_of_guests: request.body.number_of_guests,
+            created_date: request.body.created_date,
+            contact_phonenumber: request.body.contact_phonenumber,
+            contact_name: request.body.contact_name,
+            contact_email: request.body.contact_email,
+            meal_id: request.body.meal_id,
+        })
+    response.json(insertedReservations)
+
+})
+
+
+router.get("/:id", async (request, response) => {
+    const reservationId = request.params.id;
```

Comment:
would recommend to use a try catch block where you capture the error in case the insert fails and return an response to the client as an error code